sklearn/preprocessing/imputation.py

```diff
@@ -298,6 +396,80 @@ def _dense_fit(self, X, strategy, missing_values, axis):
 
             return most_frequent
 
+        # KNN
+        elif strategy == "knn":
+            if self.copy:
+                X = np.copy(X)
+
+            if axis == 1:
+                X = X.transpose()
+                mask = mask.transpose()
+
+            #Get dimensions and missing count
```

Comment:
PEP8 requires space between # and comment. But I also don't think this comment helps.